### PR TITLE
Fix: added sequelize to create and manage postgres db

### PR DESCRIPTION
**Pull request by sbelete01**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+const database = '<postgres db name>'
+const password = '<postgres password>'
+const port = '<postgres port>'
+
+import { Sequelize, Model, DataTypes } from 'sequelize'
+// require(sequelize)
+
+const db = new Sequelize({
+    database: 'flick_db',
+    username: 'samuelbelete',
```
Comment (badams92) [inline database/index.js]: let's save username and stuff as `.env` variables

Comment (carlbenjaminlyon) [reply database/index.js]: With Ben on this.

**Comment by carlbenjaminlyon** [comment]
Needs a table for movies

**Comment by badams92** [comment]
looks good

**Comment by carlbenjaminlyon** [comment]
Looks good!

**Pull request merged**